docs/src/main/paradox/reference.md

```diff
@@ -0,0 +1,64 @@
+# Reference
+
+This is the reference documentation for an Alpakka connector. This section should contain
+a general overview of the connector and mention the libraries and APIs that the connector
+is using. Also it should link to external resources that might help to learn
+about the technology connector is using.
+
+### Reported issues
+
+[Tagged issues at Github](https://github.com/akka/alpakka/labels/p%3Areference)
+
+## Artifacts
+
+@@dependency [sbt,Maven,Gradle] {
+  group=com.lightbend.akka
+  artifact=akka-stream-alpakka-reference_$scalaBinaryVersion$
+  version=$version$
+}
+
+## Usage
+
+Give a brief description of the usage of this connector. If you want to mention a
+class name, make sure to link to the API docs:
+@scaladoc[ReferenceReadMessage](akka.stream.alpakka.reference.ReferenceReadMessage).
+
+### Reading messages
+
+If any of the API classes are different between Scala and Java, link to both API docs:
+@scala[@scaladoc[Reference](akka.stream.alpakka.reference.scaladsl.Reference$)]
+@java[@scaladoc[Reference](akka.stream.alpakka.reference.javadsl.Reference$)].
+
+Show an example code snippet of how a source of this connector can be created.
+
+Scala
+: @@snip [snip]($alpakka$/reference/src/test/scala/docs/scaladsl/ReferenceSpec.scala) { #source }
```

Comment:
Add text underneath to show it is flipped with the selected language.

Reply:
Done.

Reply:
Ah, I start to believe you are not aware of this feature. Adding text after a snippet without an empty line will flip in the same tab.

Reply:
Ahh, I see. Never used it like this.